frontend/src/metabase/components/SaveQuestionForm/context.tsx

```diff
@@ -82,9 +82,23 @@ export const SaveQuestionProvider = ({
     }
   }, [isLoading]);
 
-  const lastUsedDashboard = recentItems?.find(
-    item => item.model === "dashboard",
-  ) as RecentCollectionItem | undefined;
+  const lastUsedDashboard = useMemo(() => {
```

Comment:
the `lastUsedDashboard` variable name doesn't capture the "ignore the dashboard if there's a more recent collection" part. maybe `defaultDashboard` is a better name?